apps/docs/scripts/search/generate-embeddings.ts

```diff
@@ -193,19 +191,14 @@ async function generateEmbeddings() {
         const input = content.replace(/\n/g, ' ')
 
         try {
-          const configuration = new Configuration({ apiKey: process.env.OPENAI_KEY })
-          const openai = new OpenAIApi(configuration)
+          const openai = new OpenAI({ apiKey: process.env.OPENAI_KEY })
 
-          const embeddingResponse = await openai.createEmbedding({
+          const embeddingResponse = await openai.embeddings.create({
             model: 'text-embedding-ada-002',
             input,
           })
 
-          if (embeddingResponse.status !== 200) {
```

Comment:
This error will still be handled by the catch block on line 220 (although without `inspect`). Should I add `inspect` there/

Reply:
Should be okay now without the `inspect` 👍